rct229/rulesets/ashrae9012019/section22/section22rule12.py

```diff
@@ -0,0 +1,106 @@
+            is_primary_rule=True,
+            rmr_context="ruleset_model_descriptions/0",
+            list_path="$.heat_rejections[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_b = context.baseline
+        heat_rejection_loop_ids_b = (
+            get_heat_rejection_loops_connected_to_baseline_systems(rmd_b)
+        )
+
+        return heat_rejection_loop_ids_b
+
+    def create_data(self, context, data):
+        rmd_b = context.baseline
+        heat_rejection_loop_ids_b = (
+            get_heat_rejection_loops_connected_to_baseline_systems(rmd_b)
+        )
+
+        number_of_baseline_heat_rejections_b = sum(
+            [
+                1
+                for heat_rejection_b in find_all("$.heat_rejections[*]", rmd_b)
+                if getattr_(heat_rejection_b, "heat_rejections", "loop")
+                in heat_rejection_loop_ids_b
+            ]
+        )
+
+        return {
+            "heat_rejection_loop_ids_b": heat_rejection_loop_ids_b,
+            "number_of_baseline_heat_rejections_b": number_of_baseline_heat_rejections_b,
+        }
+
+    class HeatRejectionRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section22Rule12.HeatRejectionRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+            )
+
+        def get_calc_vals(self, context, data=None):
+            heat_rejection_loop_ids_b = data["heat_rejection_loop_ids_b"]
+            number_of_baseline_heat_rejections_b = data[
+                "number_of_baseline_heat_rejections_b"
+            ]
+            return {
+                "heat_rejection_loop_ids_b": heat_rejection_loop_ids_b,
+                "number_of_baseline_heat_rejections_b": number_of_baseline_heat_rejections_b,
+            }
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            heat_rejection_loop_ids_b = calc_vals["heat_rejection_loop_ids_b"]
+            number_of_baseline_heat_rejections_b = calc_vals[
+                "number_of_baseline_heat_rejections_b"
+            ]
+
+            return (
+                number_of_baseline_heat_rejections_b == 1
+                and len(heat_rejection_loop_ids_b) == 1
+            )
+
+        def get_fail_msg(self, context, calc_vals=None, data=None):
+            heat_rejection_loop_ids_b = calc_vals["heat_rejection_loop_ids_b"]
+            number_of_baseline_heat_rejections_b = calc_vals[
+                "number_of_baseline_heat_rejections_b"
+            ]
+            len_heat_rejection_loop_ids_b = len(heat_rejection_loop_ids_b)
+
+            if (
+                number_of_baseline_heat_rejections_b == 1
+                and len_heat_rejection_loop_ids_b > 1
+            ):
+                FAIL_MSG = "There is more than one condenser loop for this building. There should only be one condenser loop attached to all chillers in the baseline chiller plant."
```

Comment:
This should not be the building, but the project.

Let's update the message to:
"There is more than one condenser loop for this project. There should only be one condenser loop attached to all chillers in the baseline chiller plant."

Reply:
Addressed.